README.md

```diff
@@ -1,36 +1,115 @@
 # silverstripe-encrypt-at-rest
 
-This module allows data to be encrypted in the database, but be decrypted when extracted from the database, using a
-secret key (hopefully) known only by the web server.
+This module allows Silverstripe CMS ORM data to be encrypted before being stored in the database, and automatically decrypted before using within your application. To do this, we use a secret key known only by the web server.
 
-*Note:* This does not provide significant protection except in the case of database compromise. It should be used as
```

Comment:
This is a significant caveat and should not be removed

Reply:
This is okay - it's been expanded into a larger bullet point further down (under the new 'Caveats to understand' section).